packages/experimental/TabList/src/TabListAnimatedIndicator/useAnimatedIndicatorStyles.ts

```diff
@@ -35,15 +76,24 @@ export function useAnimatedIndicatorStyles(): AnimatedIndicatorStyles {
     if (vertical) {
       containerStyles.start = x + tabBorderWidth + 1;
       indicatorStyles.top = y + startMargin + tabBorderWidth + 1;
+      indicatorStyles.transform = [{ translateY: indicatorTranslate as any }, { scaleY: indicatorScale as any }];
```

Comment:
also below on 83

Reply:
TSLint throws a fit when using the Animated.Value type as a value for a transform on a View, even though it's valid. Should I just suppress the error here?

Reply:
hmmm thinking about it a bit more it may be because the `transform` prop itself is expected to be the animated value, not its component props -- natively animations are wired from inputs combined/interpolated to an output prop (e.g. `transform` or `opacity`), so I'd be a little concerned that `Animated` can't handle this structure with the native driver and you may be getting dropped down to the JS animation driver.  I don't know the best way to drop a debug breakpoint to clarify if the JS or native animation driver is being used to make the final change to the `transform` prop on the View; @Saadnajmi would you be able to lend a quick pointer to where the `transform` prop update is handled that we could see the callstack and be sure of which animated driver is being used?

Reply:
Eh... If it's just a transform, I think I can point to where that's set. If it's transform + Animated, it gets messy with typing and I'm not sure. Entirely possible the types are wrong, and I would mostly just test it it works 😅

Reply:
ah, `Animated.AnimatedProps<ViewStyle>`...